icu4j/main/classes/core/src/com/ibm/icu/message2/PlainStringFormattedValue.java

```diff
@@ -15,7 +15,7 @@
  * Expect it to change.
  *
  * @internal ICU 72 technology preview. Visible For Testing.
- * @deprecated This API is for ICU internal use only.
+ * @deprecated This API is for technology preview only.
```

Comment:
So... this class is only public to be "Visible For Testing". It's probably better if it doesn't say "technology preview" at all. @mihnita

Reply:
PlainStringFormatterValue is a class implementing com.ibm.icu.text.FormattedValue. This is the concrete class used by the message formatter2. Although we have a design option not to expose such class as public API for ICU users, it implies our user cannot create an equivalent object to the output of the formatter.

If we really want to hide this, we don't even need to say "technology preview". If our intent is only for ICU internal testing, I'd drop the term.

I think "Visible For Testing" is good for user experimenting the technology preview MessgeFormatter. So I prefer to keep "technology preview only".

Reply:
I don't want to block the release candidate, but we are not planning for this to eventually become public.
Ok for rc. I am asking @mihnita to change this class back to truly `@internal` right after rc.

I think these should say
```
 * @internal Visible For Testing.
 * @deprecated This API is for unit testing only.
 *
```

Reply:
OK. It's still @internal and deprecated comment should not affect signature data file.